lib/roo_on_rails/default.env

```diff
@@ -9,3 +9,8 @@ NEW_RELIC_DEVELOPER_MODE=false
 NEW_RELIC_LICENSE_KEY=override-me
 RACK_SERVICE_TIMEOUT=10
 RACK_WAIT_TIMEOUT=30
+SIDEKIQ_ENABLED=true
+SIDEKIQ_THREADS=5
+SIDEKIQ_DB_POOL_SIZE=6
+SIDEKIQ_DB_REAPING_FREQUENCY=10
```

Comment:
Or we could reduce the default. I'm quite unconvinced that 25 threads makes sense for the majority of use cases; but either way, a low default might protect us a bit until we do have `pg_bouncer` set up everywhere.